### DISCUSSION TITLE
Why sometime hovering on contribution section didn't show tooltip of numbers of contribution!

### DISCUSSION THREAD

**Discussion by Vishal-beep136** [community/community#31752] [Other Feature Feedback, Questions, & Ideas]
## Hey! Does This Is Happening to me only -
Whenever I scroll to bottom in contributions section and try to hover in a square to see contribution It doesn't show tooltip of how many contributions made on that particular date. I have to refresh the page to make that work!.

**Comment by radar-blip** [comment]
Its not just you, its a small glitch happening to many.

**Reply by Vishal-beep136** [reply]
I see

**Comment by jfullstackdev** [comment]
is this fixed by now ? mine is working by now

**Reply by Vishal-beep136** [reply]
Didn't fixed for me yet!

**Comment by Enyium** [comment]
Are y'all's problems related to #155899, or is this something different?
